src/bitcoin/mod.rs

```diff
@@ -196,6 +221,8 @@ impl Config {
             capacity_limit: 100 * 100_000_000, // 100 BTC
             #[cfg(not(feature = "testnet"))]
             capacity_limit: 21 * 100_000_000, // 21 BTC
+            fee_pool_target_balance: 10_000_000, // 0.1 BTC
```

Comment:
used as usat value in `give_rewards` (so 10_000_000_000_000 would be 0.1 BTC)

Reply:
Addressed by changing the check in `give_rewards` rather than changing the config value (since other config values are in sats).